libs/ui-watt/src/lib/components/input/datepicker/watt-datepicker.spec.ts

```diff
@@ -344,6 +344,30 @@ describe('Datepicker', () => {
           )
         ).toBeInTheDocument();
       });
+
+      it('should update control value when only end date has changed', async () => {
+        await setup(withInitialValue);
+
+        const [, range] = screen.getAllByRole('button', {
```

Comment:
what does the comma mean in the const [, range] ?

Reply:
`screen.getAllByRole` will return two results. The first result will be the single datepicker and the second will be the datepicker with range. We could have written it like this `const [single, range] = screen.getAllByRole('button'` however since we don't use the singe datepicker for anything, we would then have an unused variable so by just writing the `,` tells that we just ignore the first "result" which is the single datepicker.

Reply:
You can find more examples here: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment#examples